barchartrealtime.html

```diff
@@ -0,0 +1,41 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+ <title>Learning D3</title>
+  <style>
```

Comment:
Please can we put all of the styles into a CSS file so that the HTML only contains the content